packages/web3-react/src/types.ts

```diff
@@ -15,6 +15,28 @@ GNU Lesser General Public License for more details.
 You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
+import { ReactNode } from 'react'
+
 export type Theme = 'auto' | 'web95' | 'retro' | 'soft' | 'midnight' | 'minimal' | 'rounded' | 'nouns'
 export type Mode = 'light' | 'dark' | 'auto'
 export type CustomTheme = any // TODO: define type
+
+export type Connector = {
+  id: string
```

Comment:
Should we introduce a type for this id too?

Reply:
You mean an enum or sth similar? Sounds like a good idea 👍

Reply:
Yes, but I am not sure whether it's doable due to injected connector